packages/demos/src/pagination/PaginationBasic.vue

```diff
@@ -1,7 +1,14 @@
 <script setup>
-  import Pagination from '@/components/pagination/pagination.vue';
+  import { AcvPagination } from '@acronis-platform/ui-component-library';
```

Comment:
that import will come from dist. 
Its not a problem in production, but you did that on purpose?

Reply:
I did this because I prefer to have an import with a reference to a component, instead of a reference to a file type (global.d.ts). 

but objectively my link doesn't lead to the file itself either, I can change this if I need to

Reply:
No need if you dont want to use it that way

Reply:
in constructor we are using imports like 
import { AcvPagination } from '@acronis-platform/ui-component-library';

Reply:
reverted back